src/qt/clientmodel.cpp

```diff
@@ -90,14 +104,6 @@ int64_t ClientModel::getHeaderTipTime() const
     return cachedBestHeaderTime;
 }
 
-void ClientModel::updateTimer()
-{
-    // no locking required at this point
-    // the following calls will acquire the required lock
-    Q_EMIT mempoolSizeChanged(m_node.getMempoolSize(), m_node.getMempoolDynamicUsage());
-    Q_EMIT bytesChanged(m_node.getTotalBytesRecv(), m_node.getTotalBytesSent());
-}
```

Comment:
moving this code into a lambda is stylistic only, right?

Reply:
I could keep this here but the connection from the timer needs to be direct (not auto) so that this would run in the timer's slot.

Also, this is now unreachable, and it's a small step towards removing polling.